yarn-project/aztec/src/cli/aztec_start_options.ts

```diff
@@ -0,0 +1,280 @@
+      description: 'Deploys L1 Aztec contracts before starting the node. Needs mnemonic or private key to be set',
+      defaultValue: 'false',
+      envVar: 'DEPLOY_AZTEC_CONTRACTS',
+    },
+    {
+      flag: '--node.publisherPrivateKey <value>',
+      description: 'Private key of account for publishing L1 contracts',
+      defaultValue: undefined,
+      envVar: 'L1_PRIVATE_KEY',
+    },
+    {
+      flag: '--node.l2QueueSize <value>',
+      description: 'Size of queue of L2 blocks to store in world state',
+      defaultValue: '1000',
+      envVar: 'L2_QUEUE_SIZE',
+    },
+    {
+      flag: '--node.worldStateBlockCheckIntervalMS <value>',
+      description: 'Frequency in which to check for blocks in ms',
+      defaultValue: '100',
+      envVar: 'WS_BLOCK_CHECK_INTERVAL_MS',
+    },
+  ],
+  'P2P SUBSYSTEM': [
+    {
+      flag: '--p2p-enabled',
+      description: 'Enable P2P subsystem',
+      defaultValue: 'false',
+      envVar: 'P2P_ENABLED',
+    },
+    ...getOptions('p2p', p2pConfigMappings),
+  ],
+  TELEMETRY: [
+    {
+      flag: '--telemetry',
+      description: 'Enable telemetry',
+      defaultValue: 'false',
+      envVar: 'TELEMETRY',
+    },
+    ...getOptions('tel', telemetryClientConfigMappings),
+  ],
+  PXE: [
+    {
+      flag: '--pxe',
+      description: 'Starts Aztec PXE with options',
+      defaultValue: undefined,
+      envVar: undefined,
+    },
+    ...getOptions('pxe', pxeCliConfigMappings),
+  ],
+  ARCHIVER: [
+    {
+      flag: '--archiver',
+      description: 'Starts Aztec Archiver with options',
+      defaultValue: undefined,
+      envVar: undefined,
+    },
+    // filter out archiverUrl as it's passed separately in --node & --prover-node
+    ...getOptions('archiver', archiverConfigMappings).filter(opt => !opt.flag.includes('archiverUrl')),
+  ],
+  SEQUENCER: [
+    {
+      flag: '--sequencer',
+      description: 'Starts Aztec Sequencer with options',
+      defaultValue: undefined,
+      envVar: undefined,
+    },
+    ...getOptions('sequencer', sequencerClientConfigMappings),
+  ],
+  'PROVER AGENT': [
+    {
+      flag: '--prover',
+      description: 'Starts Aztec Prover Agent with options',
+      defaultValue: undefined,
+      envVar: undefined,
+    },
+    ...getOptions('prover', proverClientConfigMappings),
+  ],
+  'PROVER NODE': [
+    {
+      flag: '--prover-node',
+      description: 'Starts Aztec Prover Node with options',
+      defaultValue: undefined,
+      envVar: undefined,
+    },
+    {
+      flag: '--proverNode.archiverUrl <value>',
+      description: 'URL for an archiver service',
+      defaultValue: undefined,
+      envVar: 'ARCHIVER_URL',
+    },
+    ...getOptions(
+      'proverNode',
+      // filter out archiver options from prover node options as they're passed separately in --archiver
+      filterConfigMappings(proverNodeConfigMappings, Object.keys(archiverConfigMappings) as (keyof ArchiverConfig)[]),
```

Comment:
Maybe this means we should remove the `archiver` options from the `proverNodeConfig`? It was my bad including them in there.